### ISSUE TITLE
Will it have monads in the future?

### ISSUE THREAD

**Issue by alekseysavchuk** [Chubru/micro-lang#1]
I find this project very promising. I'm into functional programming🤓 from my youth, and in this language I personally miss monads🥰. Just wondering, it'll have monads in the near future.

**Comment by Chubru** [comment]
Good idea. However, at the moment, I’m very busy because I’m rewriting the compiler from the outdated and slow language `Go` to `micro-lang`. This should result in a significant performance boost. 
Can you help me with implementing monads?
